napari/utils/transforms/transforms.py

```diff
@@ -499,6 +540,11 @@ def scale(self) -> npt.NDArray:
         self._setup_decompose_linear_matrix_cache()
         return self._cache_dict['decompose_linear_matrix'][1]
 
+    @property
+    def voxel_size(self) -> tuple[pint.Quantity, ...]:
+        """Return the voxel size of the transform."""
+        return tuple(x * y for x, y in zip(self.scale, self.units))
```

Comment:
I don't like voxel_size because it's not semantically very different from "scale" — it doesn't immediately tell me that it's got units attached. Also, I must admit I was confused by the use of "x * y" (the variable names sound like axis names) in this context. 😅 So I propose:

```suggestion
    @property
    def physical_scale(self) -> tuple[pint.Quantity, ...]:
        """Return the scale of the transform, with units."""
        return tuple(magnitude * unit for magnitude, unit in zip(self.scale, self.units))
```

Reply:
Alternative:

```suggestion
    @property
    def physical_scale(self) -> tuple[pint.Quantity, ...]:
        """Return the scale of the transform, with units."""
        return tuple(np.multiply(self.scale, self.units))
```